scripts/config/testnet.config.ts

```diff
@@ -14,5 +14,6 @@ export default {
         EraManager: [60 * 60], // 1 day
         ServiceAgreementRegistry: [1e6], //threshold
         PurchaseOfferMarket: [1e5, '0x0000000000000000000000000000000000000000'],
+        IndexerRegistry: ['1000000000000000000000'],
```

Comment:
Why not use `1000e18`, I think it would be more readable

Reply:
1e18 will cause overflow to convert to BigNumber